examples/splittext_puttcp_config.json

```diff
@@ -0,0 +1,86 @@
+{
+    "rootGroup": {
+        "name": "MiNiFi Flow",
+        "processors": [
+            {
+                "name": "Get files from /tmp/input",
+                "identifier": "a8fd96f8-6f3e-4ab8-9cc7-7dcb3304d60b",
+                "type": "org.apache.nifi.processors.standard.GetFile",
+                "schedulingStrategy": "TIMER_DRIVEN",
+                "schedulingPeriod": "2 sec",
+                "penaltyDuration": "30 sec",
+                "yieldDuration": "1 sec",
+                "properties": {
+                    "Input Directory": "/tmp/input"
+                },
+                "autoTerminatedRelationships": []
+            },
+            {
+                "name": "Split text by lines skipping the first 3 header lines",
+                "identifier": "40b45f52-ae1d-443c-9465-7e070ce3bea0",
+                "type": "org.apache.nifi.processors.standard.SplitText",
+                "schedulingStrategy": "EVENT_DRIVEN",
+                "schedulingPeriod": "1 sec",
```

Comment:
I think we should remove the scheduling period setting from EVENT_DRIVEN processors.

Reply:
Updated in dcab8ffd0883e3fc881e85f6a30fcbdf0cb7e9f2